HIP/hip-867.md

```diff
@@ -0,0 +1,119 @@
+---
+hip: 867
+title: Add Smart Contract Services Support for KZG Point Evaluation Precompiled Function
+author: Danno Ferrin (@shemnon)
+working-group: Nana Essilfie-Conduah <@nana-ec>. David Bakin <@david-bakin-sl>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: 2024-01-24
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/872
+---
+
+## Abstract
+
+Add the Ethereum KZG Point Evaluation Precompile from EIP-4844: Shard Blob
+Transactions.
+
+## Motivation
+
+One part of [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844) is the KZG Point
+Evaluation function, a cryptographic precompile that can be used with zero
+knowledge proofs as well as proving that certain subsections of a blob are
+present without introducing the entire blob. This precompile adds value to the
+other parts of EIP-4844, but it does not require them. Hedera will expose
+support for this precompile without introducing other parts of the EIP.
+
+## Rationale
+
+While Hedera will not be supporting most EIP-4844 features the KZG Point
+Evaluation Precompile is cleanly severable from the other integrated features.
+This precompile, when accompanied by the associated trusted setup, can be viewed
+as a cryptographic primitive similar to ECDSA. The ability to prove subsets of
+hashed data provides unique functionality not found in other cryptographic
+precompiles. Ethereum apps may use the KZG features without any interation with
+blob varying transactions, and it makes sense to allow Hedera Smart Contract
+apps the same latitude.
+
+This precompile will be brought in with the same semantics and calling
+conventions as Ethereum Mainnet. It also requires a customizable data file known
+as the trusted setup. For maximum compatibility the same trusted setup from
+Ethereum Mainnet will be used.
+
+## User stories
+
+* As a smart contract developer I want to use the EIP-4844 Point Evaluation
+  Precompile in my smart contract applications.
+
+## Specification
+
+There are no changes needed to Hedera Services and the JSON-RPC relay.
+
+### EVM Implementation
+
+The [KZG Point Evaluation precompile](https://eips.ethereum.org/EIPS/eip-4844#point-evaluation-precompile)
+will be implemented as it is in EIP-4844.
+
+The address for the precompile will be the same as Ethereum
+Mainnet (`0x000000000000000000000000000000000000000A`).
+
+The point evaluation will use the
+same [trusted setup](https://github.com/ethereum/c-kzg-4844/blob/main/src/trusted_setup.txt)
```

Comment:
https://eips.ethereum.org/EIPS/eip-4844#cryptographic-helpers points to that doc.  Since we are not implementing it but bringing in the Besu EVM which has already implemented it, I consider the reference to the EIP to be sufficient.